LyricFy/Views/Components/Home/AddProjectsCell.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  AddProjectsCell.swift
+//  LyricFy
+//
+//  Created by Marcos Costa on 04/05/23.
+//
+
+import Foundation
+import UIKit
+
+class AddProjectsCell: UICollectionViewCell {
+    
+    static let identifier: String = "AddProjectsCell"
+
+    lazy var projectComponent: UIView = {
+        let projectComponent = UIView()
+        projectComponent.translatesAutoresizingMaskIntoConstraints = false
+        projectComponent.backgroundColor = .cyan
+        projectComponent.layer.cornerRadius = 10
+        return projectComponent
+    }()
+    
+    lazy var image: UIImageView = {
+        let image = UIImageView(image: UIImage(systemName: "plus"))
+        image.translatesAutoresizingMaskIntoConstraints = false
+        return image
+    }()
+    
+    override init(frame: CGRect) {
```

Comment:
Não precisa sobrepor o init com o parâmetro frame se o frame for atribuído como .zero em toda inicialização

Reply:
feito